Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,181 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
 | Heerlen           |
 | Alkmaar           |
 +-------------------+
-28 rows in set (0.00 sec)
\ No newline at end of file
+28 rows in set (0.00 sec)
```

Comment:
Be careful about what changes make it into a PR. You probably didn't need to include the week1 homework.